plugins/s3/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.19...3.24 FATAL_ERROR)
+
+project(
+  s3
+  DESCRIPTION "S3 plugin for Tenzir"
+  LANGUAGES CXX)
+
+include(CTest)
+
+find_package(Tenzir REQUIRED)
+
+TenzirRegisterPlugin(
+  TARGET s3
+  ENTRYPOINT src/plugin.cpp)
```

Comment:
This either needs to be enabled in CI, or should become a builtin instead as previously discussed for GCS.

Reply:
I'll make this a builtin tomorrow.

Reply:
Made this a builtin now.